misk-hibernate/src/main/kotlin/misk/vitess/StartVitessService.kt

```diff
@@ -69,7 +71,8 @@ class VitessCluster(
           ?: throw IllegalStateException("vitess_schema_resource_root must be specified")
       val hasVschema = resourceLoader.walk(config.vitess_schema_resource_root)
           .any { it.endsWith("vschema.json") }
-      check(hasVschema) { "schema root not valid, does not contain any vschema.json: ${config.vitess_schema_resource_root}"}
+      check(
+          hasVschema) { "schema root not valid, does not contain any vschema.json: ${config.vitess_schema_resource_root}" }
```

Comment:
super nit: I would cut the line after the `{` instead.

Reply:
This was Intellij...